tsdate/core.py

```diff
@@ -1274,6 +1281,12 @@ def date(
         from the inside algorithm in addition to the dated tree sequence. If
         ``return_posteriors`` is also ``True``, then the marginal likelihood
         will be the last element of the tuple.
+    :param bool method_of_moments: If ``True`` match central moments in variational gamma
```

Comment:
I'm wondering if we should comment out these parameter descriptions for the moment, as well as making sure the global_prior and max_iterations are undocumented, then we can introduce them once we have decided on the best API? E.g. I wonder if the original idea of a dictionary of control parameters is more flexible? It seems a little odd to have some parameters that don't apply to the `inside-outside` method in there too.

Another possibility would be to define `tsdate.date_vgamma` and `tsdate.date_discrete` (or some such) as the base functions, and simply use `tsdate.date` as a wrapper, which passes `**kwargs` to the base functions (and then we document the specific options within the base functions).

I will ask Ben and Jerome what the best approach is here.